.github/workflows/bat.yml

```diff
@@ -170,3 +170,21 @@ jobs:
           grep "deploying" buildlog.txt
           ! grep "checking" buildlog.txt
           rm buildlog.txt
+
+      - name: Run build with startup options
+        uses: ./
+        with:
+          tasks: error deploy
+          build-options: -continueOnFailure
+          startup-options: -logfile console.log
+
+      - name: Verify workflow command is added
+        run: |
+          grep "deploy - task" console.log
+          grep "::error::" console.log
+          rm console.log
+            
+          
+
+
+
```

Comment:
The updates look good but the extra newlines are driving me nuts 😆

Reply:
done.